ts/components/leftpane/overlay/OverlayClosedGroup.tsx

```diff
@@ -9,22 +9,46 @@ import { MemberListItem } from '../../MemberListItem';
 import { OverlayHeader } from './OverlayHeader';
 // tslint:disable: no-submodule-imports use-simple-attributes
 
-import { setOverlayMode } from '../../../state/ducks/section';
+import { resetOverlayMode } from '../../../state/ducks/section';
 import { getPrivateContactsPubkeys } from '../../../state/selectors/conversations';
 import { SpacerLG } from '../../basic/Text';
 import { MainViewController } from '../../MainViewController';
 import useKey from 'react-use/lib/useKey';
+import styled from 'styled-components';
+import { SessionSearchInput } from '../../SessionSearchInput';
+import { getSearchResults, isSearching } from '../../../state/selectors/search';
+import { VALIDATION } from '../../../session/constants';
+
+const StyledMemberListNoContacts = styled.div`
+  font-family: var(--font-font-mono);
+  background: var(--color-cell-background);
+  text-align: center;
+  padding: 20px;
+`;
+
+const StyledGroupMemberListContainer = styled.div`
+  padding: 2px 0px;
+  width: 100%;
+  max-height: 400px;
+  overflow-y: auto;
+  border: var(--border-session);
+`;
+
+const NoContacts = () => {
+  return (
+    <StyledMemberListNoContacts>{window.i18n('noContactsForGroup')}</StyledMemberListNoContacts>
```

Comment:
Surprised the formatted didn't catch this. Shouldn't it be?

```tsx
<StyledMemberListNoContacts>
  {window.i18n('noContactsForGroup')}
</StyledMemberListNoContacts>
```

Reply:
there are 97 lines and prettier is set to allow 100, that's why :D

Reply:
rofl 😂